bin/release/src/release/elastic_beanstalk/.ebextensions/01_metabase.config

```diff
@@ -3,6 +3,24 @@
 # Modify the environmental variables below to customize your installation
 # Comment out a variable to disable a feature
 #####
+
+packages:
+  yum:
+    perl-DateTime: []
+    perl-Sys-Syslog: []
+    perl-LWP-Protocol-https: []
+    perl-Switch: []
+    perl-URI: []
+    perl-Digest-SHA: []
+sources: 
+  /opt/cloudwatch: https://aws-cloudwatch.s3.amazonaws.com/downloads/CloudWatchMonitoringScripts-1.2.1.zip
```

Comment:
Isn't the scripts deprecated? https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/monitoring-scripts-intro.html